packages/graphql/src/registry.ts

```diff
@@ -0,0 +1,46 @@
+import { navigateProgram, type Program, type SemanticNodeListener } from "@typespec/compiler";
+import type { GraphQLObjectType } from "graphql";
+
+type Mutable<T> = {
+  -readonly [k in keyof T]: T[k];
+};
```

Comment:
@steverice there is a version of this in `@typespec/compiler/src/utils/misc.ts` that we can probably use instead.